src/linear_algebra/multilinear/basic.lean

```diff
@@ -213,7 +213,7 @@ def of_subsingleton [subsingleton ι] (i' : ι) : multilinear_map R (λ _ : ι,
   map_smul' := λ m i r x, by
   { rw subsingleton.elim i i', simp only [function.eval, function.update_same], } }
 
-variables {M₂}
+variables (M₁) {M₂}
```

Comment:
I'm probably overlooking something obvious, but I can't see why you need `M₁` to be explicit in `const_of_is_empty`

Reply:
How else would it be inferred?

Reply:
I asked that because you seem to always use `_` to fill in this argument, which suggests that in all cases currently in mathlib the expected type is enough. But I get that in the middle of the proof it can be annoying to add a type annotation, so I'm okay with that.

Reply:
Ah, for this PR I needed the argument to state `norm_const_of_is_empty` without needing `@` or annotations